plugins/rbac/src/types.ts

```diff
@@ -17,7 +17,7 @@ export type RolesData = {
 
 export type MembersData = {
   name: string;
-  type: 'User' | 'Group';
+  type: string;
```

Comment:
Just out of curiosity why this change? Is there a value other than `'User' | 'Group'` ?